exp-player/addon/components/exp-frame-base.js

```diff
@@ -85,10 +87,11 @@ export default Ember.Component.extend({
             this.set('eventTimings', timings);
         },
         next() {
-            console.log('Leaving frame ID', this.get('id'));
+            var frameId = `${this.get('frameIndex')}-${this.get('id')}`;
```

Comment:
Is it hard to reverse the order here? Seems like the `frameIndex` is better suited as a suffix. I'm already hesitant to tamper with the id values a researcher specifies for a segment of the experiment, and a suffix seems slightly less obtrusive (this is just an opinion).